torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -352,11 +352,6 @@ def _annotate_conv2d_bn(
         if len(conv_partition.output_nodes) > 1:
             raise ValueError("conv partition has more than one output node")
         conv_node = conv_partition.output_nodes[0]
-        conv_node_users = list(conv_node.users.keys())
-        if len(conv_node_users) > 1:
-            raise ValueError(
-                "Conv node must be consumed by BN only for it to be fusable."
```

Comment:
is this change related to the rest of the PR?

Reply:
Yeah this check was added for the cpu graph, but the cuda graph has another user

Reply:
@andrewor14 you mean the cuda graph, after the bn decomp is added (in this PR), is still different from cpu graph?

Reply:
Yeah, it's just one op (forgot the name, something to do with memory format) but it's dead code and gets eliminated later anyway

Reply:
I think technically this check can't be removed since otherwise the fusion will not be valid?